addons/website_project/controllers/main.py

```diff
@@ -36,3 +38,12 @@ def insert_record(self, request, model, values, custom, meta=None):
                 message_type='comment',
             )
         return res
+
+    def _prepare_visitor_partner_vals(self, request):
+        return {
+            'email': request.params.get('email_from', False),
+            'name': request.params.get('partner_name', False),
+            'phone': request.params.get('partner_phone', False),
+            'company_name': request.params.get('partner_company_name', False),
+            'lang': request.lang.code,
+        }
```

Comment:
It does not seem, that hook is overriden somewhere, right? If yes, then the hook is useless, we could directly do the content of that method inside the method in which we call this one.

Reply:
![without_store](https://github.com/odoo/odoo/assets/98956163/e0d4d482-a50c-42d9-a486-90d835ea74e2)


if we don't store these fields it will show those values in other information.

Reply:
> ![without_store](https://private-user-images.githubusercontent.com/98956163/345727922-e0d4d482-a50c-42d9-a486-90d835ea74e2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAwNzg1MjQsIm5iZiI6MTcyMDA3ODIyNCwicGF0aCI6Ii85ODk1NjE2My8zNDU3Mjc5MjItZTBkNGQ0ODItYTUwYy00MmQ5LWE0ODYtOTBkODM1ZWE3NGUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA0VDA3MzAyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVjODBhYzRiOTJmMDdkOWE2NmQwMDY2YmQyYzI3MmE1ZmY0MmJiMDA2OWM0YzY1NDY0NWQ1Y2UzNjAwOTFhNGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YdvRWMnpIvn4pIKPdRdtvUBJI_avUbPtNJLywftuVrs)
> 
> if we don't store these fields it will show those values in other information.

let me confirm with aju

Reply:
my bad.. :see_no_evil:  no need to store it.

Reply:
> my bad.. 🙈 no need to store it.

Alright, we no longer need to confirm with the PO.